canonical-web-and-design/juju.is
# Build the new navigation

## Done

- Add the new navigation

## QA

- Go to https://juju-is-188.demos.haus
- And check the navigation behaves as it should be
- Compare the navigation with the [design](https://app.zeplin.io/project/606203889af57e12287b5308/screen/606f060bdb2a5851a7c8cc62) and [mobile design](https://app.zeplin.io/project/606203889af57e12287b5308/dashboard?sid=60794b6d1b4172b84713d715)

**Some of the links are not available yet**
**I still need to add some links to the navigation items. See the [copy](https://docs.google.com/document/d/147spDkSVH0YaEGKZ4tDi7i28zzhIcacF1hBOMHvxaC0/edit?disco=AAAAL5F2SJI)**

## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1998
